CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-master
+    * HOTFIX      #2074 [ListBuilder]       Added resolve-parameter for join-condition
```

Comment:
This doesn't say anything. Remember that the target audience of the CHANGELOG is a user of Sulu.

Can you give a description that a Sulu user understands?